docs/blog/oras-0.15-a-fully-functional-registry-client.md

```diff
@@ -0,0 +1,189 @@
+# ORAS 0.15: A fully Functional OCI Registry Client
+
+_Author: [Feynman Zhou](https://twitter.com/FeynmanZhou), [Yi Zha](https://github.com/yizha1)_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced two releases of v0.15 for the ORAS CLI recently. ORAS [v0.15.0](https://github.com/oras-project/oras/releases/tag/v0.15.0) introduces four new top-level commands and new options to manage tags and repos in registries and provides capabilities to alter the OCI content for advanced use cases. Three weeks later, [ORAS 0.15.1](https://github.com/oras-project/oras/releases/tag/v0.15.1) also released with a few known bug fixes. Since the release of v0.15, ORAS CLI has evolved into a fully functional OCI registry client.
+
+## What's new in ORAS 0.15
+
+![ What's new in ORAS 0.15](oras-0.15/what's-new-in-oras-0.15.png)
+
+As ORAS has been adopted by more and more OCI implementors and registry vendors, we have seen increased community requirements in providing fine-grained capabilities to alter the content of OCI supply chain artifacts. ORAS 0.15 now supports operate the blob and manifest of an artifact in the registry. Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.15.0) for details.
+
+This blog post will demonstrate how to use ORAS CLI v0.15 to convert a Docker image stored in Docker Hub into an OCI image， then push it to the Distribution registry.
+
+## Run a local Distribution registry
+
+Run a local instance of the CNCF Distribution Registry, with ORAS Artifacts support:
+
+```
+docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.2
+```
+
+## Fetch and view the manifest of a sample Docker image
+
+```
+$ oras manifest fetch docker.io/library/hello-world@sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4
+
+{
+   "schemaVersion": 2,
+   "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+   "config": {
+      "mediaType": "application/vnd.docker.container.image.v1+json",
+      "size": 1469,
+      "digest": "sha256:feb5d9fea6a5e9606aa995e879d862b825965ba48de054caab5ef356dc6b3412"
+   },
+   "layers": [
+      {
+         "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
+         "size": 2479,
+         "digest": "sha256:2db29710123e3e53a794f2694094b9b4338aa9ee5c40b930cb8063a1be392c54"
+      }
+   ]
+}
+```
+
+## Fetch and push a blob
```

Comment:
Am I seeing a bunch of details for how I can convert a docker image to an OCI image?
This feels a little detailed without the context for why?
And, if I am trying to convert a docker format to an OCI image format, seems we would have a `oras manifest-convert` or some such command to do this intrinsically. I think I saw a similar thing from Brandon in his regclient that merges foreign layers from a windows image to non-forien layer manifest. The scenario for both is manifest conversion. 
Is that what we're trying to show here?

Reply:
The purpose of using these command sets (`oras blob` and `oras manifest`) here is to solve a specific issue with ORAS 0.15, showcasing most of the new commands. In fact, we can regard this as a workaround for the conversion scenario. The proposal of `oras manifest-convert` could be supported in the future if we really see more conversion requirements come in from the community and solve this issue intrinsically.